luigi/parameter.py

```diff
@@ -959,3 +960,37 @@ def serialize(self, x):
         :param x: the value to serialize.
         """
         return json.dumps(x)
+
+class NumericalParameter(Parameter):
+    """Parameter that restricts values to the given numerical type, i.e. int or
+    float, and to the specified range
+    """
+    def __init__(self, var_type=int, min_value=0, max_value=maxsize,
+                 left_op=operator.lt, right_op=operator.le, *args, **kwargs):
```

Comment:
What is a `left_op`/`right_op`, can you add docs? What values can a user replace them with?